src/views/clusteroverview/SettingsTab/ClusterTab/components/GeneralSettings/TemplatesAndImagesManagement/components/AutomaticImagesDownload/AutomaticImagesDownload.tsx

```diff
@@ -28,8 +29,12 @@ const AutomaticImagesDownload: FC<AutomaticImagesDownloadProps> = ({
   const isAdmin = useIsAdmin();
 
   const [hyperConverged, loaded] = hyperConvergeConfiguration;
-  const isEnabledAutomaticImagesDownload =
-    hyperConverged?.spec?.featureGates?.enableCommonBootImageImport;
+  const isEnabledAutomaticImagesDownload = isEmpty(
+    hyperConverged?.spec?.enableCommonBootImageImport,
```

Comment:
if its not feature gate but value is null? possible?

Reply:
how should i consider null? true or false? Or unset? I think would be unset

Reply:
`Note: spec.enableCommonBootImageImport is true by default.` so should be true if its unset okay. Changing accordingly